packages/node_modules/@ractivejs/cli/src/scaffolds.js

```diff
@@ -0,0 +1,103 @@
+export const component = `
+<link rel="ractive" href="">
+
+<template>
```

Comment:
The current CLI uses `<template id="somePartial">...</template>` for partials added to the template. Is that still the case here?

Reply:
The things like dynamic CSS and other things the current CLI does is not yet included. Will add those next.